var/spack/repos/builtin/packages/openmpi/package.py

```diff
@@ -528,6 +532,11 @@ def configure_args(self):
                 '--with-valgrind={0}'.format(spec['valgrind'].prefix),
             ])
 
+        # Singularity container support
+        if spec.satisfies('@:4.9'):
+	    if '+singularity' in spec:
+	       config_args.append(
+                 '--with-singularity={0}'.format(spec['singularity'].prefix))
```

Comment:
Wouldn't a single condition:
```suggestion
        # Singularity container support
        if spec.satisfies('+singularity @:4.9'):
            singularity_opt = '--with-singularity={0}'.format(spec['singularity'].prefix)
            config_args.append(singularity_opt)
```
work as well?

Reply:
That worked, wasn't aware of this way in `spec`, thanks